bazel_example/BUILD.bazel

```diff
@@ -1,4 +1,5 @@
 load("@com_googleapis_gapic_generator_csharp//rules_csharp_gapic:csharp_gapic.bzl", "csharp_proto_library", "csharp_grpc_library", "csharp_gapic_library")
+load("@com_googleapis_gapic_generator_csharp//rules_csharp_gapic:csharp_gapic_pkg.bzl", "csharp_gapic_assembly_pkg")
 
 proto_library(
     name = "example_src",
```

Comment:
I should have noticed naming like this in the original reveiw (sorry)
According to bazel conventions `proto_library` targets must be called `"<api>_proto"`. Please rename this to `example_proto`

Reply:
done